pkg/printers/internalversion/printers_test.go

```diff
@@ -1271,7 +1271,7 @@ func TestPrintPod(t *testing.T) {
 					},
 				},
 			},
-			[]metav1.TableRow{{Cells: []interface{}{"test6", "1/2", "NotReady", "6", "<unknown>"}}},
+			[]metav1.TableRow{{Cells: []interface{}{"test6", "1/2", "Running", "6", "<unknown>"}}},
```

Comment:
This test case pod status should be `Running` according https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/internalversion/printers.go#L841